packages/react/src/experimental/Collections/visualizations/collection/Table/index.tsx

```diff
@@ -61,6 +65,8 @@ export const TableCollection = <
     Sortings
   >(source)
 
+  console.log(data)
```

Comment:
delete?

Reply:
sorry. I need a lint rule, i always forgot the console.logs in the code